cmd/protoc-gen-validate-cpp/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"github.com/envoyproxy/protoc-gen-validate/module"
+	pgs "github.com/lyft/protoc-gen-star"
+	"google.golang.org/protobuf/types/pluginpb"
+)
+
+func main() {
+	optional := uint64(pluginpb.CodeGeneratorResponse_FEATURE_PROTO3_OPTIONAL)
+	pgs.
+		Init(pgs.DebugEnv("DEBUG_PGV"), pgs.SupportedFeatures(&optional)).
+		RegisterModule(module.ValidatorForLanguage("cc")).
```

Comment:
This is the main change (along with removing `RegisterPostProcessor(pgsgo.GoFmt()).` for non-Go generators).